arturo.gemspec

```diff
@@ -1,33 +1,34 @@
-# my_rails_engine.gemspec
-Gem::Specification.new do |gem|
-  gem.version = '2.2.1'
-  gem.name = 'arturo'
-  gem.files = Dir["lib/**/*", "app/**/*", "config/**/*"] + %w(README.md CHANGELOG.md LICENSE)
-  gem.summary = "Feature sliders, wrapped up in an engine"
-  gem.description = "Deploy features incrementally to your users"
-  gem.license = "APLv2"
-  gem.email = "james.a.rosen@gmail.com"
-  gem.homepage = "http://github.com/jamesarosen/arturo"
-  gem.authors = ["James A. Rosen"]
-  gem.test_files = []
-  gem.require_paths = [".", "lib"]
-  gem.has_rdoc = 'false'
-  gem.specification_version = 2
-  gem.required_ruby_version = '>= 2.0'
+require_relative 'lib/arturo/version'
 
-  private_key_path = File.expand_path("~/.ssh/rubgems/arturo-private_key.pem")
-  gem.signing_key = private_key_path if File.exists?(private_key_path)
-  gem.cert_chain = [ "gem-public_cert.pem" ]
+Gem::Specification.new do |s|
+  s.name = 'arturo'
+  s.version = Arturo::VERSION
+  s.authors = ['James A. Rosen']
+  s.email = 'james.a.rosen@gmail.com'
 
-  gem.add_runtime_dependency      'activerecord', '> 3.2', '< 5.1'
-  gem.add_development_dependency  'rails',        '> 3.2', '< 5.1'
-  gem.add_development_dependency  'mocha',        '~> 1.1'
-  gem.add_development_dependency  'rake',         '~> 10.3'
-  gem.add_development_dependency  'minitest',     '> 0', '< 6.0'
-  gem.add_development_dependency  'minitest-rg',  '> 0', '< 6.0'
-  gem.add_development_dependency  'rails-dom-testing', '~> 1.0'
-  gem.add_development_dependency  'sqlite3',      '~> 1.3'
-  gem.add_development_dependency  'factory_girl', '~> 4.2'
-  gem.add_development_dependency  'timecop',      '~> 0.3'
-  gem.add_development_dependency  'wwtd',         '~> 0.5'
+  s.summary = 'Feature sliders, wrapped up in an engine'
+  s.homepage = 'http://github.com/zendesk/arturo'
+  s.files = Dir['lib/**/*', 'app/**/*', 'config/**/*'] + %w(README.md CHANGELOG.md LICENSE)
+  s.description = 'Deploy features incrementally to your users'
+  s.require_paths = ['.', 'lib']
+
+  s.license = 'APLv2'
+  private_key_path = File.expand_path('~/.ssh/rubgems/arturo-private_key.pem')
```

Comment:
yep

On Fri, May 13, 2016 at 2:05 PM, Pierre Schambacher <
notifications@github.com> wrote:

> In arturo.gemspec
> <https://github.com/zendesk/arturo/pull/99#discussion_r63251056>:
>
> > -  gem.add_development_dependency  'rake',         '~> 10.3'
> > -  gem.add_development_dependency  'minitest',     '> 0', '< 6.0'
> > -  gem.add_development_dependency  'minitest-rg',  '> 0', '< 6.0'
> > -  gem.add_development_dependency  'rails-dom-testing', '~> 1.0'
> > -  gem.add_development_dependency  'sqlite3',      '~> 1.3'
> > -  gem.add_development_dependency  'factory_girl', '~> 4.2'
> > -  gem.add_development_dependency  'timecop',      '~> 0.3'
> > -  gem.add_development_dependency  'wwtd',         '~> 0.5'
> > +  s.summary = 'Feature sliders, wrapped up in an engine'
> > +  s.homepage = 'http://github.com/zendesk/arturo'
> > +  s.files = Dir['lib/**/*', 'app/**/*', 'config/**/*'] + %w(README.md CHANGELOG.md LICENSE)
> > +  s.description = 'Deploy features incrementally to your users'
> > +  s.require_paths = ['.', 'lib']
> > +
> > +  s.license = 'APLv2'
> > +  private_key_path = File.expand_path('~/.ssh/rubgems/arturo-private_key.pem')
>
> Should I just kill it?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/arturo/pull/99/files/3e3edc689c307cd30169e1e69156e1c5d3292401..b9e6757e339727c908e67ff54b81100afa436da3#r63251056>
>